### DISCUSSION TITLE
Admin-Dashboard: Active Users and Interaction are suddenly empty

### DISCUSSION THREAD

**Discussion by weingaertnerit** [mastodon/mastodon#18431] [Q&A]
Hi at all,

where can I troubleshoot this. All Services are running and no errors are logged, but the statistic is empty since 3 or 4 hours: 

![grafik](https://user-images.githubusercontent.com/42450475/168661020-c2a9d90a-db29-4ea2-8efd-c409e722f613.png)

Thank you very much

**Comment by nemobis** [comment]
I don't have an answer but I've got a similar question about how the dashboard is updated. https://github.com/mastodon/mastodon/discussions/18663

This probably has to do with some cache. It might be a mix of Redis caching and Sidekiq jobs but I'm not sure.
